### PR TITLE
Climber feature

### PR DESCRIPTION
**Pull request by GaiaZano05**

### CODE REVIEW

**Review by barel13 — APPROVED** [review]

**Review by barel13 — APPROVED** [review]
WOW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -49,6 +49,9 @@
 
         public static final double STOP_CLIMBER = 149.5; // [s]
 
+        public static final double SIMULATION_LOOP_PERIOD = 0.02; // [s]
```
Comment (katzuv) [inline src/main/java/frc/robot/Constants.java:52]: Isn't there already a "loop period" constant?

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -49,6 +49,9 @@
...
+
```
Comment (katzuv) [inline src/main/java/frc/robot/Constants.java:53]: One line is enough.
```suggestion
```

**Review by katzuv — CHANGES_REQUESTED** [review]
Rename the branch's name to `feature/climber` ([here](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch)'s how to do that).

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (barel13) [reply src/main/java/frc/robot/Constants.java:52]: not for the simulations

**Comment by barel13** [comment]
I think it shouldn't be in Climber's constants, its related also to other subsystems

**Pull request closed without merging**